city_scrapers/spiders/det_employment_solutions.py

```diff
@@ -0,0 +1,153 @@
+        "MWDB": "div.mwdb-meeting_wrapper_inner-wrapper",
+        # TODO: find selector for CEAC meetings
+        "CEAC": "div",
+    }
+
+    def set_current_meeting_type(self, item):
+        """Setter function for curr_meeting_type property."""
+        if not item:
+            self.curr_type = None
+        info = item.get()
+        # Right now there are only 2 parsable meeting types on the webpage
+        if "mwdb" in info:
+            self.curr_type = self.MEETING_TYPES[0]
+        if "CEAC" in info:
+            self.curr_type = self.MEETING_TYPES[1]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        NOTE: The entire monthly schedules are ONLY available for download
+        in a docx and pdf format.
+
+        Parsable meeting types:
+         - MWDB Meetings
+         - CAEC Meetings
+
+        Unparsable meetings types (e.g. PDF/DOCX):
+         - DESC Meetings
+        """
+        meeting_info_sections = response.css("section.desc-public-meeting")
+        # XXX: Assumes location is the same across meeting types.
+        location = self._parse_location(meeting_info_sections[0])
+
+        for items in meeting_info_sections[1:]:
+            self.set_current_meeting_type(items)
+            for meeting_item in items.css(
+                self.MEETING_TYPE_TO_CSS_SELECTOR_MAP[self.curr_type]
+            ):
+                logging.debug(meeting_item.get())
+                meeting = Meeting(
+                    title=self._parse_title(items),
+                    description=self._parse_description(items),
+                    classification=self._parse_classification(items),
+                    start=self._parse_start(meeting_item),
+                    end=self._parse_end(meeting_item),
+                    all_day=False,
+                    time_notes=self._parse_time_notes(items),
+                    location=location,
+                    links=self._parse_links(items),
+                    source=response.url,
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title"""
+        if self.curr_type == "MWDB":
+            return "Mayor's Workforce Development Board (MWDB) Meeting"
+        if self.curr_type == "CEAC":
+            return "Career and Education Advisory Council (CEAC) Meeting"
+        return "DESC Meeting"
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        # TODO
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        if self.curr_type == "MWDB":
+            return BOARD
+        if self.curr_type == "CEAC":
+            return ADVISORY_COMMITTEE
+        return NOT_CLASSIFIED
```

Comment:
Usually we'll want to default to one of the existing types instead of using `NOT_CLASSIFIED`, since there's typically one that's the majority